cli/tools/standalone.rs

```diff
@@ -242,3 +249,13 @@ pub fn compile_to_runtime_flags(
     watch: None,
   })
 }
+
+/// This function checks if input path has trailing slash or not. If input path
+/// has trailing slash it will return true else it will return false.
+pub fn path_has_trailing_slash(path: &Path) -> bool {
+  if let Some(path_str) = path.to_str() {
+    cfg!(windows) && path_str.ends_with('\\') || path_str.ends_with('/')
+  } else {
+    false
+  }
+}
```

Comment:
Maybe we should move this to `fs_util.rs` as there seems to be some other path methods there and then add some unit tests?